src/layouts/SiteDashboard/SiteDashboard.stories.tsx

```diff
@@ -118,4 +123,58 @@ RequestApproved.parameters = {
   },
 }
 
+export const SiteLaunchStepsCompleted = Template.bind({})
+SiteLaunchStepsCompleted.decorators = [
+  (Story) => {
+    return (
+      <MemoryRouter initialEntries={["/sites/storybook/dashboard"]}>
+        <Route path="/sites/:siteName/dashboard">
+          <SiteLaunchProvider
+            initialSiteLaunchStatus="LAUNCHED"
+            initialStepNumber={SITE_LAUNCH_TASKS_LENGTH}
+          >
+            <Story />
+          </SiteLaunchProvider>
+        </Route>
+      </MemoryRouter>
+    )
+  },
+]
+
+export const SiteLaunchLaunching = Template.bind({})
+SiteLaunchLaunching.decorators = [
+  (Story) => {
+    return (
+      <MemoryRouter initialEntries={["/sites/storybook/dashboard"]}>
+        <Route path="/sites/:siteName/dashboard">
+          <SiteLaunchProvider
+            initialSiteLaunchStatus="LAUNCHING"
+            initialStepNumber={SITE_LAUNCH_TASKS_LENGTH}
+          >
+            <Story />
+          </SiteLaunchProvider>
+        </Route>
+      </MemoryRouter>
+    )
+  },
+]
+
+export const SiteLaunchPartialStepsCompleted = Template.bind({})
+SiteLaunchPartialStepsCompleted.decorators = [
+  (Story) => {
+    return (
+      <MemoryRouter initialEntries={["/sites/storybook/dashboard"]}>
+        <Route path="/sites/:siteName/dashboard">
+          <SiteLaunchProvider
+            initialSiteLaunchStatus="CHECKLIST_TASKS_PENDING"
+            initialStepNumber={1}
```

Comment:
not a problem wtih the story, but if the `stepNumber` is implicitly tied to the status, then we shouldn't expose them individually

Reply:
Hey, while they are related, but it is not possible to derive the step number from the siteLaunchStatus!
The checklist step number state is solely held within the frontend for now!